### PR TITLE
Update UG to include addnote, editnote and deletenote features

### PR DESCRIPTION
**Pull request by wang-jun-hao**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -93,69 +97,126 @@ Example:
...
 
-### 3.8 Adding a medical note: `note`
+### 3.8 Adding a medical note: `addnote`
 
 Adds a medical note to a patient.
 
-As a doctor, you can add a medical note to a patient when viewing his/her profile. MediBook will automatically add the 
-to the patient displayed on the screen.
+As a doctor, you can add a medical note to your patient when viewing his/her profile to keep track of consultations
+with your patients. MediBook will automatically add the medical note to the patient displayed on your screen.
```
Comment (foojingyi) [inline docs/UserGuide.md]: Should this be a user story in the UG? May not be relevant to some readers of the UG. Is there a better way to phrase this/refer to doctors themselves? i.e. If you are a doctor...

File: docs/UserGuide.md [hunk 2]
```diff
@@ -93,69 +97,126 @@ Example:
...
 when on the main list and then `access`-ing the patient's index on the filtered list.
+* `MCR` refers to your medical registration number. It is in the format 'M@####$', where @ is a letter/digit, # is a digit and $ is a letter.
+* You do not need to include your title (Dr) in your name.
 * For your convenience, the date field can be omitted. MediBook will automatically select today's date.
-* If you specify the date of the medical note, it has to be in the past or today.
+* If you specify the date of the medical note, it cannot be in the future.
+
+####Example:
+
+Context: while viewing the profile page of patient with IC 'S9123456A'
```
Comment (foojingyi) [inline docs/UserGuide.md]: Can this sound more user centric? i.e. If you are viewing the profile of a patient with the IC...

**Review by foojingyi — COMMENTED** [review]
Can be more user-centric?

File: docs/UserGuide.md (hunk 1 above)
Comment (wang-jun-hao) [reply docs/UserGuide.md]: Ok I have updated this.

File: docs/UserGuide.md (hunk 2 above)
Comment (wang-jun-hao) [reply docs/UserGuide.md]: Ok updated

**Review by PrestonTYR — APPROVED** [review]
lgtm

**Pull request merged**